src/app.js

```diff
@@ -1,21 +1,58 @@
+/* eslint-disable no-console */
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
+const fs = require('fs');
+const path = require('path');
+
+function moveFile() {
+  const [enteredSourcePath, enteredDestinationPath] = process.argv.slice(2);
+  const fileName = path.basename(enteredSourcePath);
+  const basePath = path.resolve(__dirname, `../`);
```

Comment:
Do you really need to move upper for build basePath? Why do you need this?

Reply:
Yes I agree, it was unnecessary to create not only the basePath, but also to resolve enteredSourcePath and enteredDestinationPath (I can just use a relative path)